app/di.scala

```diff
@@ -3,9 +3,15 @@ import com.gu.atom.data._
 import com.gu.atom.publish._
 import data._
 import com.gu.pandahmac.HMACAuthActions
+import util.{LogShipping, LogShippingImpl}
+
 
 class Module extends AbstractModule {
   def configure() = {
+
+    bind(classOf[LogShipping])
+      .to(classOf[LogShippingImpl]).asEagerSingleton()
```

Comment:
1. It sounds cool
2. https://www.playframework.com/documentation/2.5.x/ScalaDependencyInjection#Eager-bindings made it seem like that was what I wanted